ui/raidboss/data/07-dt/raid/r8s.ts

```diff
@@ -1,14 +1,1907 @@
+      },
+    },
+    {
+      id: 'R8S Terrestrial Rage Spread/Stack',
+      // For Shadowchase (A3BC), actors available roughly 2.9s after cast
+      // Only need one of the 5 actors to determine pattern
+      // Ids are sequential, starting 2 less than the boss
+      // Two patterns (in order of IDs):
+      // S, WSW, NW, NE, ESE
+      // N, ENE, SE, SW, WNW
+      // TODO: Add orientation call?
+      type: 'HeadMarker',
+      netRegex: { id: [headMarkerData.stack, headMarkerData.spread], capture: false },
+      condition: (data) => data.phase === 'rage',
+      delaySeconds: 0.1,
+      suppressSeconds: 1,
+      infoText: (data, _matches, output) => {
+        if (data.hasSpread)
+          return data.isFirstRage ? output.spreadThenStack!() : output.spreadBehindClones!();
+
+        if (data.stackOnPlayer === data.me)
+          return data.isFirstRage
+            ? output.stackThenSpread!({
+              stack: output.stackOnYou!(),
+            })
+            : output.stackOnYouBehindClones!();
+
+        if (data.stackOnPlayer !== undefined) {
+          const name = data.party.member(data.stackOnPlayer);
+          return data.isFirstRage
+            ? output.stackThenSpread!({
+              stack: output.stackOnPlayer!({ player: name }),
+            })
+            : output.stackOnPlayerBehindClones!({ player: name });
+        }
+      },
+      run: (data) => {
+        data.stackOnPlayer = undefined;
+        data.hasSpread = undefined;
+        data.isFirstRage = false;
+      },
+      outputStrings: {
+        spreadThenStack: Outputs.spreadThenStack,
+        stackThenSpread: {
+          en: '${stack} => Spread',
+        },
+        spreadBehindClones: {
+          en: 'Spread (Behind Clones)',
+        },
+        stackOnPlayer: Outputs.stackOnPlayer,
+        stackOnPlayerBehindClones: {
+          en: 'Stack on ${player} (Behind Clones)',
+        },
+        stackOnYou: Outputs.stackOnYou,
+        stackOnYouBehindClones: {
+          en: 'Stack on YOU (Behind Clones)',
+        },
+      },
+    },
+    {
+      id: 'R8S Shadowchase Rotate',
+      // Call to move behind Dragon Head after clones dash
+      type: 'StartsUsing',
+      netRegex: { id: 'A3BD', source: 'Howling Blade', capture: true },
+      delaySeconds: (_data, matches) => parseFloat(matches.castTime),
+      suppressSeconds: 1,
+      infoText: (_data, _matches, output) => {
+        return output.rotate!();
+      },
+      outputStrings: {
+        rotate: {
+          en: 'Rotate',
+        },
+      },
+    },
+    {
+      id: 'R8S Weal of Stone',
+      // TODO: Call direction that the heads are firing from, needs OverlayPlugin
+      type: 'StartsUsing',
+      netRegex: { id: 'A78E', source: 'Wolf of Stone', capture: false },
+      suppressSeconds: 1,
+      infoText: (_data, _matches, output) => {
+        return output.lines!();
+      },
+      outputStrings: {
+        lines: {
+          en: 'Lines',
```

Comment:
Just `Lines` is a bit ambiguous. Feel free to use other wording e.g. `dodge` or whatever instead.

```suggestion
          en: 'Avoid Lines',
```